Dockerfile

```diff
@@ -1,145 +1,163 @@
 #--------------------------------------
 # Target image to build
 #--------------------------------------
-ARG IMAGE=latest
+ARG TARGET=latest
 
 #--------------------------------------
 # Ubuntu base image to use
 #--------------------------------------
-ARG FLAVOR=latest
+ARG BASE_IMAGE=ubuntu:bionic
 
 #--------------------------------------
-# renovate rebuild trigger
+# Default customizable args
 #--------------------------------------
-FROM renovate/ubuntu@sha256:b20bd58a962952580352f731606c63f621e525cc4706a4b78baf71e852b69da4 as base-latest
-FROM renovate/ubuntu:bionic@sha256:1ab4cb684944bfd76b2a29c368e4e221b796f08fb826c1a826327476cf196647 as base-bionic
-FROM renovate/ubuntu:focal@sha256:b20bd58a962952580352f731606c63f621e525cc4706a4b78baf71e852b69da4 as base-focal
+ARG USER_NAME=user
+ARG USER_ID=1000
+ARG APP_ROOT=/usr/src/app
 
 #--------------------------------------
 # Image: base
 #--------------------------------------
-FROM base-${FLAVOR} as base
+FROM ${BASE_IMAGE} as base
 
-ARG BUILDPACK_VERSION
-LABEL org.opencontainers.image.source="https://github.com/renovatebot/docker-buildpack" \
-  org.opencontainers.image.version="${BUILDPACK_VERSION}"
+ARG USER_NAME
+ARG USER_ID
+ARG APP_ROOT
 
-USER root
+LABEL maintainer="Rhys Arkins <rhys@arkins.net>" \
+  org.opencontainers.image.source="https://github.com/renovatebot/docker-buildpack"
 
-# Zombie killer: https://github.com/Yelp/dumb-init#readme
-RUN apt-get update && \
-  apt-get install -y dumb-init && \
-  rm -rf /var/lib/apt/lists/*
-
-# loading env
+#  autoloading buildpack env
 ENV BASH_ENV=/usr/local/etc/env
-SHELL ["/bin/bash" , "-c"]
+SHELL ["bash" , "-c"]
+
+ENTRYPOINT ["docker-entrypoint.sh"]
 
-ENTRYPOINT [ "docker-entrypoint.sh" ]
+ENV DEBIAN_FRONTEND noninteractive
+ENV LC_ALL C.UTF-8
+ENV LANG C.UTF-8
```

Comment:
Moving this to our internal env? to cleanup some more?

Reply:
As part of `install-buildpack`?

Reply:
yes, good idea